<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Climate Forecasting Toolbox</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/cft.ico</normaloff>icon/cft.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="zoningButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Zoning"/>
    </property>
    <property name="text">
     <string>Zoning</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fcstButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Forecasting"/>
    </property>
    <property name="text">
     <string>Forecasting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>130</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Verification"/>
    </property>
    <property name="text">
     <string>Verification</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
